lib/cn/omf.h

```diff
@@ -298,8 +302,10 @@ struct vblock_footer_omf {
     uint32_t vbf_rsvd;
 } HSE_PACKED;
 
-static_assert(sizeof(struct vblock_footer_omf) <= (VBLOCK_FOOTER_LEN - (2 * HSE_KVS_KEY_LEN_MAX)),
-              "Vblock footer overflow");
+/* Storing 2 keys in the footer: min and max. */
+#define VBLOCK_FOOTER_PAGES \
```

Comment:
Currently we don't support a vblock footer spanning across multiple pages. That's what the earlier static_assert() verified.

Reply:
Thanks for pointing that out. I'll add that back in.